.github/workflows/remote-benchmarks.yml

```diff
@@ -0,0 +1,72 @@
+# Run benchmarks on a remote system for the release the project is using
+# - This workflow is reusable with run-type set to release or nightly
+# - Scripts ending in "-local" run on the github runner
+# - Scripts ending in "-remote" run are executed on the remote test server through ssh
+
+name: Benchmark Test on Docker Deephaven
+
+on:
+  workflow_call:
+    inputs:
+      run-type:
+        required: true
+        type: string
+
+jobs:
+  process-benchmarks:
+
+    runs-on: ubuntu-22.04
+    env:
+      SD: .github/scripts
+      HOST: ${{secrets.BENCHMARK_HOST}}
+      USER: ${{secrets.BENCHMARK_USER}}
+      REPO: ${{github.repository}}
+      BRANCH: ${{github.ref_name}}
+      RUN_TYPE: ${{inputs.run-type}}
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Local and Remote Scripts
+      run: |
+        sudo chmod +x ${SD}/*
+        ${SD}/setup-ssh-local.sh ${HOST} "${{secrets.BENCHMARK_KEY}}"
+
+    - name: Run Remote Test Server Setup
+      run: |
+        ${SD}/run-ssh-local.sh ${HOST} ${USER} ${SD} setup-test-server-remote ${REPO} ${BRANCH} ${RUN_TYPE}
+
+    - name: Run Remote Benchmark Artifact Build
+      run: |
+        ${SD}/run-ssh-local.sh ${HOST} ${USER} ${SD} build-benchmark-artifact-remote
+
+    - name: Run Remote Benchmarks
+      run: |
+        ${SD}/run-ssh-local.sh ${HOST} ${USER} ${SD} run-benchmarks-remote
+        
+    - name: Fetch Benchmark Results
+      run: |
+        ${SD}/fetch-results-local.sh ${HOST} ${USER} ${RUN_TYPE}
+        
+    - name: Remove Summary Results
+      run: |
+        # For now remove any summaries before putting in GCloud
+        rm -f ${RUN_TYPE}/*.csv
+        
+    - name: Authorize GCloud Credentials
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{secrets.BENCHMARK_GCLOUD}}
```

Comment:
This is a different secret name in `deephaven/benchmark`.

Reply:
I have no permission to see the secrets (even the names) on deephaven/benchmark.  In any case, they need to be clear.  I currently have:
- BENCHMARK_GCLOUD
- BENCHMARK_HOST
- BENCHMARK_KEY
- BENCHMARK_USER
Are you saying you have:
- DEEPHAVEN_BENCHMARK_KEY
- BENCHMARK_HOST
- BENCHMARK_KEY
- BENCHMARK_USER
If so, that doesn't really distinguish between the GCloud Json key and the Private Key for SSH on the GH runner.  I'm fine with changing all of them to be clearer.  I just need to know what they are.

Reply:
Good point. I'll update the secret to be named `BENCHMARK_GCLOUD`